mypy.ini

```diff
@@ -342,7 +342,7 @@ ignore_errors = True
 [mypy-scipy.signal.windows._windows]
 ignore_errors = True
 
-[mypy-scipy.signal._bsplines]
+[mypy-scipy.signal._spline_filters]
```

Comment:
The changes related to the rename (`_bsplines` -> `_spline_filters`) should move to the commit that does the rename, i.e. https://github.com/scipy/scipy/commit/48dfec69357e1f0983426ddd76e295e077cfa452